### PR TITLE
Soumyo/ec2 routes

### PR DESCRIPTION
**Pull request by soumyo13**
### Description

With the route table resource, the route resource from the ec2 is developed. This is done on a existing resource. In the docs all are mentioned.

### Issues Resolved
NA

List any existing issues this PR resolves, or any Discourse or StackOverflow discussion that's relevant

### Check List
Please fill box or appropriate ([x]) or mark N/A.
- [x] New functionality includes integration tests/controls
- [x] New Terraform resources
- [x] Documentation provided or updated for resources 
- [x] All Integration Tests pass
- [x] All Unit Tests pass
- [x] `rake lint` passes
- [x] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

File: docs/resources/aws_route_table.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ platform: aws
 
 # aws\_route\_table
 
-Use the `aws_route_table` InSpec audit resource to test properties of a single Route Table. A route table contains a set of rules, called routes, that are used to determine where network traffic is directed.
+Use the `aws_route_table` InSpec audit resource to test properties of a single Route Table. A route table contains a set of rules, called routes, that are used to determine where network traffic is directed. This particular resource also test properties of a Route.
```
Comment (IanMadd) [inline docs/resources/aws_route_table.md:8]:
```suggestion
Use the `aws_route_table` InSpec audit resource to test the properties of a single route or route table. A route table contains a set of rules, called routes, that are used to determine where network traffic is directed.
```

File: docs/resources/aws_route_table.md [hunk 2]
```diff
@@ -67,6 +84,11 @@ See also the [AWS documentation on Route Tables](https://docs.aws.amazon.com/vpc
       its('propagating_vgws') { should be_empty }
     end
 
+##### Confirm that the route table has expected destination_cidr_block of the route
```
Comment (IanMadd) [inline docs/resources/aws_route_table.md:87]:
```suggestion
##### Confirm that the route table has the expected destination_cidr_block of the route
```

File: docs/resources/aws_route_tables.md [hunk 3]
```diff
@@ -2,13 +2,13 @@
 title: About the aws_route_tables Resource
 ---
 
-# aws\_route\_table
+# aws\_route\_tables
 
-Use the `aws_route_tables` InSpec audit resource to test properties of all or a group of Route Tables. A Route Table contains a set of rules, called routes, that are used to determine where network traffic is directed.
+Use the `aws_route_tables` InSpec audit resource to test properties of all or a group of Route Tables. A Route Table contains a set of rules, called routes, that are used to determine where network traffic is directed. This particular resource also test properties of all the Routes.
```
Comment (IanMadd) [inline docs/resources/aws_route_tables.md:7]:
```suggestion
Use the `aws_route_tables` InSpec audit resource to test properties of all or a group of route tables, or the properties of all routes. A route table contains a set of rules, called routes, that are used to determine where network traffic is directed.
```

File: libraries/aws_auto_scaling_group.rb [hunk 4]
```diff
@@ -34,11 +34,15 @@ def initialize(opts = {})
       @tags = []
 
       auto_scaling_group[:tags].map { |tag|
-        @tags.push({ 'resource_id':         tag.resource_id,
```
Comment (sathish-progress) [inline libraries/aws_auto_scaling_group.rb:37]: please remove this

**Comment by sathish-progress** [comment]
also add docs link to readme

**Comment by soumyo13** [comment]
> also add docs link to readme

done

**Comment by soumyo13** [comment]
> also add docs link to readme

done

**Review by Nirbhay1997 — APPROVED** [review]
LGTM

**Pull request merged**